.github/workflows/pypi.yml

```diff
@@ -0,0 +1,55 @@
+# This is a basic workflow to help you get started with Actions
+
+name: PYPI
+
+on:
+  # Only trigger this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      python-version: 3.7
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.python-version }}    
+      - name: Install pypa/build
+        run: |
+          python -m pip install build --user
+      - name: Check current directory
+        run: |
+           pwd
+           ls
+      - name: Set up node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Build react.js application and move to package path
+        run: |
+          npm install
+          npm run build
```

Comment:
We are using `yarn` to build packages directly and we have `yarn.lock` in our source. Maybe we should stick with Yarn for this PR.